source/css/_common/components/header/bookmark.styl

```diff
@@ -1,15 +1,20 @@
 .book-mark-link {
   border-bottom: none;
-  color: #222;
-  display: block;
-  font-size: 32px !important;
+  color: unquote(hexo-config('bookmark.color'));
+  display: inline-block;
   left: 30px;
   position: fixed;
   top: -10px;
   transition: .3s;
 
   +tablet-mobile() {
-    display: none !important;
+    display: none;
+  }
+  &::before {
+    content: "\f02e";
+    font-family: $font-family-icons;
```

Comment:
here needs color

Reply:
Why

Reply:
if not, hover color is `#222` not my custom color